seed_branding/update-generated-images.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
```

Comment:
Please rewrite it to duckscript + create a new task in https://github.com/seed-rs/seed-rs.org/blob/master/crate/Makefile.toml + make sure `cargo make` installs all necessary dependencies and it works on all platforms (Linux, Mac, Windows).
Task with Duckscript example: https://seed.discourse.group/t/replace-node-in-html-document/65/6?u=martinkavik

I'll test it then on my machine - Win 10.

And can we replace inkscape, pngcrush etc. with Rust libs (if exist)?

Reply:
Hi @MartinKavik, this is mostly a suggested workflow, not a final automation script.

1. I searched for Rust tools, but I only found `svgcleaner`
2. there are steps not easy to automate, like changing the svg viewBox and scaling the image

Given the workflow is Linux centric, I only added a `.sh` file, because the overhead of generalizing the script more is not worth the expected frequency we are going to use it.

I can perhaps remove the script, given the almost 0 utility it can have for a Windows user.

Reply:
I allow only cross-platform scripts in Seed projects. Your choice.

Reply:
Converted the script in a REAME to document my workflow. Most tools are cross platform, but I ignore the details.